src/main/java/core/basesyntax/dao/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.dao.impl;
+
+import core.basesyntax.dao.DataConverter;
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
+        removeSpaces(inputReport);
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (int i = 1; i < inputReport.size(); i++) {
+            String[] parts = inputReport.get(i).split(",");
+            if (parts.length != 3) {
+                continue;
+            }
+            FruitTransaction.Operation operation;
+            try {
+                operation = FruitTransaction.Operation.fromCode(parts[0]);
+            } catch (IllegalArgumentException e) {
+                System.err.println("Invalid operation code: " + parts[0]);
```

Comment:
Don't ignore exceptions in try-catch blocks. Instead of printing the stack trace or the message, rethrow a RuntimeException with an informative message. For example: throw new RuntimeException("Invalid operation code: " + parts[0], e);